nipype/interfaces/freesurfer/tests/test_auto_LTAConvert.py

```diff
@@ -14,23 +14,27 @@ def test_LTAConvert_inputs():
     ),
     in_fsl=dict(argstr='--infsl %s',
     mandatory=True,
-    xor=('in_lta', 'in_fsl', 'in_mni', 'in_reg', 'in_niftyreg'),
+    xor=('in_lta', 'in_fsl', 'in_mni', 'in_reg', 'in_niftyreg', 'in_itk'),
     ),
     in_lta=dict(argstr='--inlta %s',
     mandatory=True,
-    xor=('in_lta', 'in_fsl', 'in_mni', 'in_reg', 'in_niftyreg'),
+    xor=('in_lta', 'in_fsl', 'in_mni', 'in_reg', 'in_niftyreg', 'in_itk'),
     ),
     in_mni=dict(argstr='--inmni %s',
     mandatory=True,
-    xor=('in_lta', 'in_fsl', 'in_mni', 'in_reg', 'in_niftyreg'),
+    xor=('in_lta', 'in_fsl', 'in_mni', 'in_reg', 'in_niftyreg', 'in_itk'),
     ),
     in_niftyreg=dict(argstr='--inniftyreg %s',
     mandatory=True,
-    xor=('in_lta', 'in_fsl', 'in_mni', 'in_reg', 'in_niftyreg'),
+    xor=('in_lta', 'in_fsl', 'in_mni', 'in_reg', 'in_niftyreg', 'in_itk'),
     ),
     in_reg=dict(argstr='--inreg %s',
     mandatory=True,
-    xor=('in_lta', 'in_fsl', 'in_mni', 'in_reg', 'in_niftyreg'),
+    xor=('in_lta', 'in_fsl', 'in_mni', 'in_reg', 'in_niftyreg', 'in_itk'),
+    ),
+    in_reg=dict(argstr='--initk %s',
```

Comment:
This should be `in_itk`. This file really shouldn't be edited by hand; run `make specs` and add the changes. (`make specs` does modify other files, so only commit the ones with changes.)

Reply:
Whoops. Sorry, should have seen the comment on the first line. This kind of stuff is a bit new to me. :)